tests/test_modeling_wav2vec2.py

```diff
@@ -1337,3 +1336,26 @@ def test_inference_emotion_recognition(self):
 
         self.assertListEqual(predicted_ids.tolist(), expected_labels)
         self.assertTrue(torch.allclose(predicted_logits, expected_logits, atol=1e-2))
+
+    @require_pyctcdecode
+    def test_wav2vec2_with_lm(self):
+        ds = load_dataset("common_voice", "es", split="test", streaming=True)
+        sample = next(iter(ds))
+
+        import torchaudio.functional as F
```

Comment:
Should we add a require_torch_speech decorator above as well, just in case?

Reply:
yes definitely!